### ISSUE TITLE
GLPI 10 trad issues

### ISSUE THREAD

**Issue by Sp4m4** [ticgal/one-timesecret#41]
Hello, 

Everything was fine with GLPI 9.5 to use our custom fr_FR.po and fr_FR.Mo. 
Since we have migrated to GLPI 10 the custom FR files are not taken. It's the original one beginning by "Salut". 

Even we replace the original fr.po and fr.mo by our customs it is still not working. 
We tried to uninstall plugin and delete file on glpi server but same issue. 

is it a current issue ? 

thansk a lot

**Comment by OscarBeiro** [comment]
Hi,

Translations are handled here: https://localazy.com/p/one-time-secret-glpi
I cannot find any string starting with Salut.

Could you review the string at localazy and share some screenshots about the actual issue?

Regards,

**Comment by Sp4m4** [comment]
When we migrated GLPI to version 10 the FR translation files were reset to their original version when we installed the plugin. (one year ago) 
At the time it was "Salut" for "Hi" instead of "bonjour". 

GLPI's version upgrade, and probably that of One Time secret via the market place, doesn't take into account the previous language files and replaces them with a new version. 

If I put back the files I had saved, they were still knocked out because they weren't taken into account. 

I had to take the new fr_FR.po file, modify it and generate a fr_FR.Po file. 

everything's OK now, thanks for your reply.
